resources/sass/app.scss

```
@@ -79,7 +80,6 @@ nav.tabs {
         border-bottom: 1px solid #A2A9B1;
 
         li {
-            padding: 4px 10px;
             border: 2px solid white;
             transition-property: border;
```

```suggestion
            transition-property: border-bottom;
```
Without this change, my Chrome browser flashes the borders around the nav tabs upon reload. You can see it more clearly when you set the transition-duration to 1000ms (for testing).

---

Thanks for the suggestion. I will commit it as part of other changes I am adding.

---

Seems the transitions firing onload is a Chrome bug. Applying [this solution](https://stackoverflow.com/questions/14389566/stop-css-transition-from-firing-on-page-load/42969608#42969608) works.